sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -1777,6 +1777,23 @@ object functions {
   def round(e: Column, scale: Int): Column = withExpr { Round(e.expr, Literal(scale)) }
 
   /**
+   * Returns the value of the column `e` rounded to 0 decimal places with HALF_EVEN round mode.
+   *
+   * @group math_funcs
+   * @since 2.0.0
+   */
+  def bround(e: Column): Column = bround(e, 0)
```

Comment:
Thank you for review, @davies .
According to your comments, I created [SPARK-14639](https://issues.apache.org/jira/browse/SPARK-14639) for it.